storage/EtcCrypttab.cc

```diff
@@ -246,8 +246,18 @@ namespace storage
 
 	    if (boost::starts_with(blk_device, DEV_DIR "/"))
 	    {
-		if (system_info.getCmdUdevadmInfo(blk_device).get_majorminor() == majorminor)
-		    return entry;
+		try
+		{
+		    if (system_info.getCmdUdevadmInfo(blk_device).get_majorminor() == majorminor)
+			return entry;
+		}
+		catch (const Exception& e)
```

Comment:
Just a question: Would it be possible to catch just some specific exceptions? I mean this could potentially hide some real errors.

Reply:
The command udevadm may not exist. That could be handled by first catching CommandNotFoundException and rethrowing that exception. But since udevadm is use at so many places no probing will work without it so handling this here seems over-engineered to me.

Reply:
OK.